services/simcore/docker-compose.yml

```diff
@@ -539,6 +539,9 @@ services:
 
   efs-guardian:
     hostname: "{{.Service.Name}}"
+    networks:
+      - public
```

Comment:
Why do we need public network here?

Reply:
true, let me remove it :--) left over from debugging I think